spec/spec_helper_acceptance.rb

```diff
@@ -1,45 +1,35 @@
-require 'beaker-rspec'
-require 'beaker-puppet'
-require 'beaker/puppet_install_helper'
-require 'beaker/module_install_helper'
+require 'voxpupuli/acceptance/spec_helper'
```

Comment:
```suggestion
require 'voxpupuli/acceptance/spec_helper_acceptance'
```

Now that I see it, perhaps it was redundant to add `_acceptance` in the filename given the directory name.

Reply:
true, but that keeps it in sync with the spec_helper_acceptance.rb that we've in our modules. So maybe not too bad.

Reply:
I'll keep this in mind for a 1.0 release.